Makefile

```diff
@@ -763,13 +763,13 @@ edpm_deploy_baremetal: input edpm_deploy_baremetal_prep ## installs the dataplan
 .PHONY: edpm_wait_deploy_baremetal
 edpm_wait_deploy_baremetal: edpm_deploy_baremetal ## waits for dataplane readiness. Runs prep step in advance. Set DATAPLANE_REPO and DATAPLANE_BRANCH to deploy from a custom repo.
 	$(eval $(call vars,$@,dataplane))
-	oc kustomize ${DEPLOY_DIR} | oc wait --for condition=Ready --timeout=$(BAREMETAL_TIMEOUT) -f -
+	oc kustomize ${DEPLOY_DIR} | yq '. | select(.kind == "OpenStackDataPlaneNodeSet"), select(.kind == "OpenStackDataPlaneDeployment")' | oc wait --for condition=Ready --timeout=$(BAREMETAL_TIMEOUT) -f -
```

Comment:
Should we add `yq` to the dev-setup's `download_tools` role?

Reply:
I've added it recently https://github.com/openstack-k8s-operators/install_yamls/commit/eecf8bc83022ea129324fccc73aedb0bfa2ddcb3